kedify-agent/templates/keda-certs-secret.yaml

```diff
@@ -1,6 +0,0 @@
-apiVersion: v1
-kind: Secret
-type: Opaque
-metadata:
-  name: kedaorg-certs
-  namespace: {{ .Release.Namespace }}
```

Comment:
this secret is owned and managed by KEDA, adding it to kedify-agent chart would break helm upgrades - https://github.com/kedify/charts/pull/336#discussion_r2619583219. Removing it from here and only keeping it within KEDA chart